src/Profiles.ts

```diff
@@ -104,15 +105,40 @@ export class Profiles { // Processing stops if there are no profiles detected
         return this.allProfiles;
     }
 
-    public validateUrl = (newUrl: string) => {
+    public validateAndParseUrl = (newUrl: string): IUrlValidator => {
+
+        let url: URL;
+        const validProtocols: string[] = ["https"];
+        const DEFAULT_HTTPS_PORT: number = 443;
```

Comment:
No need to implement this now, but what if someone wanted to have a different default port?
Should this be a setting in our set of vscode configurable items?

Reply:
Should we allow the default port to be handled by nodejs depending on the protocol?